Error during test collection

After `pip install -e .[all]` I run into this error at test collection:

```
$ ~/munka/devel/regions [hacking|…1] $ pytest
================================================= test session starts ==================================================
platform darwin -- Python 3.7.5, pytest-5.3.5, py-1.8.0, pluggy-0.13.0

Running tests with Astropy version 4.2rc1.
Running tests in regions docs.

Date: 2020-11-08T00:14:49

Platform: Darwin-17.7.0-x86_64-i386-64bit

Executable: /usr/local/opt/python/bin/python3.7

Full Python Version: 
3.7.5 (default, Nov  1 2019, 02:16:38) 
[Clang 10.0.0 (clang-1000.11.45.5)]

encodings: sys: utf-8, locale: UTF-8, filesystem: utf-8
byteorder: little
float info: dig: 15, mant_dig: 15

Package versions: 
Numpy: 1.18.4
Scipy: 1.4.1
Matplotlib: 3.3.2
h5py: 2.10.0
Pandas: 1.1.2

Using Astropy options: remote_data: none.

rootdir: /Users/bsipocz/munka/devel/regions, inifile: setup.cfg, testpaths: regions, docs
plugins: forked-1.0.2, hypothesis-5.5.4, arraydiff-0.3, requests-mock-1.7.0, dependency-0.4.0, xonsh-0.9.13.dev1, openfiles-0.4.0, filter-subpackage-0.1.1, doctestplus-0.5.0, astropy-header-0.1.2, cov-2.8.1, remotedata-0.4.0.dev0
collected 1019 items / 1 error / 1018 selected                                                                         

======================================================== ERRORS ========================================================
__________________________ ERROR collecting regions/io/crtf/new_casa_tests/casa_mask_test.py ___________________________
ImportError while importing test module '/Users/bsipocz/munka/devel/regions/regions/io/crtf/new_casa_tests/casa_mask_test.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
regions/io/crtf/new_casa_tests/casa_mask_test.py:7: in <module>
    from ..write import write_crtf
E   ImportError: attempted relative import with no known parent package
=================================================== warnings summary ===================================================

```

--- a/regions/io/crtf/new_casa_tests/casa_mask_test.py
+++ b/regions/io/crtf/new_casa_tests/casa_mask_test.py
@@ -4,9 +4,9 @@ import pytest
 
 from astropy import coordinates, units as u
 
-from ..write import write_crtf
+from regions.io.crtf.write import write_crtf
 
-from ....shapes.ellipse import EllipseSkyRegion
+from regions.shapes.ellipse import EllipseSkyRegion
 
 try:
     from casatools import image, simulator

--- a/regions/io/crtf/new_casa_tests/casa_mask_test.py
+++ b/regions/io/crtf/new_casa_tests/casa_mask_test.py
@@ -4,9 +4,9 @@ import pytest
 
 from astropy import coordinates, units as u
 
-from regions.io.crtf.write import write_crtf
+from ..write import write_crtf
 
-from regions.shapes.ellipse import EllipseSkyRegion
+from ....shapes.ellipse import EllipseSkyRegion
 
 try:
     from casatools import image, simulator

